libs/common/include/launchdarkly/bindings/c/logging/log_level.h

```diff
@@ -23,6 +23,8 @@ enum LDLogLevel {
     LD_LOG_INFO = 1,
     LD_LOG_WARN = 2,
     LD_LOG_ERROR = 3,
+    LD_LOG_MAXVAL = 0x7FFFFFFF /* This is not a valid log level, it is used to
```

Comment:
Why is it so hard to find confirmation that this actually works on the internet?